include/sway/input/switch.h

```diff
@@ -5,6 +5,8 @@
 
 struct sway_switch {
     struct sway_seat_device *seat_device;
```

Comment:
Unrelated note: seems like this file is using mixed tabs and spaces

Reply:
Not anymore. All tabs now